src/Aquifer.API/Endpoints/Resources/Content/NeedsTranslation/List/Endpoint.cs

```diff
@@ -0,0 +1,139 @@
+        var user = await userService.GetUserFromJwtAsync(ct);
+        var (hasSearchQuery, isExactSearch) = GetSearchParams(req.SearchQuery);
+        var likeKey = isExactSearch ? "" : "%";
+        var searchParameter = new SqlParameter("SearchQuery", hasSearchQuery ? $"{likeKey}{req.SearchQuery!.Trim('"')}{likeKey}" : "");
+
+        var total = (await dbContext.Database.SqlQueryRaw<int>(BuildQuery(req, user, true, hasSearchQuery, isExactSearch), searchParameter)
+            .ToListAsync(ct)).Single();
+        var resourceContent = total == 0
+            ? []
+            : await dbContext.Database
+                .SqlQueryRaw<ResourceContentResponse>(BuildQuery(req, user, false, hasSearchQuery, isExactSearch), searchParameter)
+                .ToListAsync(ct);
+
+        var response = new Response
+        {
+            ResourceContents = resourceContent,
+            Total = total
+        };
+
+        await SendOkAsync(response, ct);
+    }
+
+    private static string BuildQuery(Request req, UserEntity user, bool getTotalCount, bool hasSearchQuery, bool isExactSearch)
+    {
+        const string selectCount = "SELECT COUNT(DISTINCT(RC.Id)) AS Count";
+        const string selectProperties = """
+                                        SELECT RC.Id AS Id, R.EnglishLabel, PR.DisplayName AS ParentResourceName,
+                                        COALESCE(RCV.SourceWordCount, RCV.WordCount) AS WordCount
+                                        """;
+
+        var groupingWithOffset = $"""
+                                  GROUP BY RC.Id, R.EnglishLabel, PR.DisplayName, L.EnglishDisplay, COALESCE(RCV.SourceWordCount, RCV.WordCount), R.SortOrder
+                                  ORDER BY PR.DisplayName, R.SortOrder, R.EnglishLabel
+                                  OFFSET {req.Offset} ROWS FETCH NEXT {req.Limit} ROWS ONLY
+                                  """;
+
+        return $"""
+                {(getTotalCount ? selectCount : selectProperties)}
+                FROM ResourceContents RC
+                    INNER JOIN Resources R ON R.Id = RC.ResourceId
+                    INNER JOIN ParentResources PR ON PR.id = R.ParentResourceId
+                    INNER JOIN Languages L ON L.Id = RC.LanguageId
+                    INNER JOIN ResourceContentVersions RCV ON RCV.ResourceContentId = RC.Id
+                WHERE RC.MediaType != {(int)ResourceContentMediaType.Audio}
+                {ApplyLanguageIdFilter(user.LanguageId ?? 1)}
+                {ApplyParentResourceIdFilter(req.ParentResourceId)}
+                {ApplySearchQueryFilter(hasSearchQuery, isExactSearch)}
+                {ApplyBookAndChapterFilter(req.BookCode, req.StartChapter, req.EndChapter)}
+                {(getTotalCount ? "" : groupingWithOffset)}
+                """;
+    }
+
+    private static string ApplyLanguageIdFilter(int languageId)
+    {
+        return $"""
+                AND RC.LanguageId = 1 AND RCV.IsPublished = 1 AND NOT EXISTS (
```

Comment:
> `RC.LanguageId = 1`

Is this supposed to be hardcoded to 1?

Reply:
1 is always English, and in this case we base needing a translation off the existence of a published English resource.